jenkins/branches/master.yml

```diff
@@ -3,10 +3,10 @@ regions:
   - oregon-b
   - frankfurt
 apps:
-  - basket-dev
+  # - basket-dev
```

Comment:
only remaining question: do we want to stop dev deployments with this PR? or should we wait until we have `basket-config` successfully doing that deploy?

Reply:
yes, let's go ahead and stop dev deployments with this PR, then we can iterate on a `.gitlab-ci.yml` in the basket-config repo that automatically deploys to dev.